### PR TITLE
Add clarification on colors with open markers

### PR DESCRIPTION
**Pull request by LiamConnors**
Add clarification on colors with open markers

### Documentation PR

- [x] I've [seen the `doc/README.md` file](https://github.com/plotly/plotly.py/blob/master/doc/README.md)
- [x] This change runs in the current version of Plotly on PyPI and targets the `doc-prod` branch OR it targets the `master` branch
- [x] If this PR modifies the first example in a page or adds a new one, it is a `px` example if at all possible
- [x] Every new/modified example has a descriptive title and motivating sentence or paragraph
- [x] Every new/modified example is independently runnable
- [x] Every new/modified example is optimized for short line count	and focuses on the Plotly/visualization-related aspects of the example rather than the computation required to produce the data being visualized
- [x] Meaningful/relatable datasets are used for all new examples instead of randomly-generated data where possible
- [x] The random seed is set if using randomly-generated data in new/modified examples
- [x] New/modified remote datasets are loaded from https://plotly.github.io/datasets and added to https://github.com/plotly/datasets
- [x] Large computations are avoided in the new/modified examples in favour of loading remote datasets that represent the output of such computations
- [x] Imports are `plotly.graph_objects as go` / `plotly.express as px` / `plotly.io as pio`
- [x] Data frames are always called `df`
- [x] `fig = <something>` call is high up in each new/modified example (either `px.<something>` or `make_subplots` or `go.Figure`)
- [x] Liberal use is made of `fig.add_*` and `fig.update_*` rather than `go.Figure(data=..., layout=...)` in every new/modified example
- [x] Specific adders and updaters like `fig.add_shape` and `fig.update_xaxes` are used instead of big `fig.update_layout` calls in every new/modified example
- [x] `fig.show()` is at the end of each new/modified example
- [x] `plotly.plot()` and `plotly.iplot()` are not used in any new/modified example
- [x] Hex codes for colors are not used in any new/modified example in favour of [these nice ones](https://github.com/plotly/plotly.py/issues/2192)

### CODE REVIEW

File: doc/python/marker-style.md [hunk 1]
```diff
@@ -378,6 +378,31 @@ fig.show()
 
 ```
 
+#### Open Marker Colors
+
+In the previous example, each marker has two colors, a marker color (set in Plotly Express with `color="species"`)  and a line color (set on the line with `color="DarkSlateGrey"`. When using open markers, like `"diamond-open"` in the following example, the marker always uses only the marker color. Adding a line color won't update the marker.
```
Comment (rl-utility-man) [inline doc/python/marker-style.md:383]: Thanks @LiamConnors!  Is something like the following a clearer version of the second sentence?

All open markers, like `"diamond-open"` in the following example have a transparent fill which means user can specify only one color and that marker color parameter controls the line, and any dot or cross.  For open markers, the marker color is the only user controllable color and the line color does nothing.

Comment (LiamConnors) [inline doc/python/marker-style.md:383]:
```suggestion
In the previous example, each marker has two colors, a marker color (set in Plotly Express with `color="species"`)  and a line color (set on the line with `color="DarkSlateGrey"`. All open markers, like "diamond-open" in the following example, have a transparent fill, which means you can specify only one color.  Specify this color using the marker color parameter. This controls the the outline color and any dot or cross. For open markers, the line color does nothing.
```
Hi @rl-utility-man, looks good. Thanks! Just updated it slightly to speak to the user

Comment (alexcjohnson) [reply doc/python/marker-style.md:383]: Other than the "the the" the text here looks great 😜

**Review by alexcjohnson — APPROVED** [review]
💃 after the suggested edits - thanks for adding this!

**Pull request merged**